app/code/Magento/ProductAlert/Controller/Add/Price.php

```diff
@@ -68,13 +71,14 @@ protected function isInternal($url)
     /**
      * Method for adding info about product alert price.
      *
-     * @return \Magento\Framework\Controller\Result\Redirect
+     * @return \Magento\Framework\App\ResponseInterface|Redirect|\Magento\Framework\Controller\ResultInterface
```

Comment:
I see that currently we could have only `\Magento\Framework\Controller\ResultInterface`. `Magento\Framework\App\ResponseInterface` is never returned here, `Redirect` implements `ResponseInterface`

Reply:
Good point. 

**implemented**

Thank you, @ihor-sviziev